base/loading.jl

```diff
@@ -593,7 +593,13 @@ function explicit_manifest_uuid_path(project_file::String, pkg::PkgId)::Union{No
         hash === nothing && return nothing
         # Keep the 4 since it used to be the default
         for slug in (version_slug(uuid, hash, 4), version_slug(uuid, hash))
-            for depot in DEPOT_PATH
+            _project = active_project()
```

Comment:
Why the underscores? There doesn't appear to be another local variable by this name and using leading underscores for local variables is strange.